apps/ledger-live-desktop/src/renderer/screens/settings/sections/Accounts/HiddenNFTCollections/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useCallback } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+import { unhideNftCollection, whitelistNftCollection } from "~/renderer/actions/settings";
+import { hiddenNftCollectionsSelector } from "~/renderer/reducers/settings";
+
+import { SettingsSection as Section, SettingsSectionRow as Row } from "../../../SettingsSection";
+import Box from "~/renderer/components/Box";
+import Track from "~/renderer/analytics/Track";
+
+import IconAngleDown from "~/renderer/icons/AngleDown";
+import { HiddenNftCollectionRow } from "./row";
+import { decodeCollectionId } from "@ledgerhq/live-nft-react";
+
+export default function HiddenNftCollections() {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [sectionVisible, setSectionVisible] = useState(false);
+
+  const hiddenNftCollections = useSelector(hiddenNftCollectionsSelector);
+
+  const onUnhideCollection = useCallback(
+    (collectionId: string) => {
+      dispatch(unhideNftCollection(collectionId));
+      dispatch(whitelistNftCollection(collectionId));
+    },
+    [dispatch],
+  );
+
+  const toggleHiddenCollectionsSection = useCallback(() => {
+    setSectionVisible(prevState => !prevState);
+  }, []);
+
+  return (
+    <Section style={{ flexDirection: "column" }}>
+      <Track onUpdate event="HiddenNftCollections dropdown" opened={sectionVisible} />
+      <Row
+        title={t("settings.accounts.hiddenNftCollections.title")}
+        desc={t("settings.accounts.hiddenNftCollections.desc")}
+        onClick={toggleHiddenCollectionsSection}
+        contentContainerStyle={hiddenNftCollections.length ? { cursor: "pointer" } : undefined}
+      >
+        {hiddenNftCollections.length ? (
+          <Collections horizontal alignItems="center">
+            <Box ff="Inter" fontSize={3} mr={2}>
+              {t("settings.accounts.hiddenNftCollections.count", {
+                count: hiddenNftCollections.length,
+              })}
+            </Box>
+            <Show visible={sectionVisible}>
+              <IconAngleDown size={24} />
+            </Show>
+          </Collections>
+        ) : null}
+      </Row>
+
+      {sectionVisible && (
+        <Body>
+          {hiddenNftCollections.map(collectionId => {
+            const { accountId, contractAddress } = decodeCollectionId(collectionId);
+            return (
+              <HiddenNftCollectionRow
+                key={collectionId}
+                accountId={accountId}
+                contractAddress={contractAddress}
+                onUnhide={() => onUnhideCollection(collectionId)}
+              />
+            );
+          })}
+        </Body>
+      )}
+    </Section>
+  );
+}
+
+// Styled components and layout
+
+const Body = styled(Box)`
```

Comment:
Suggestion (optional) : can you move this on top of this component in order to apply  `no-use-before-define`